src/lib/types/openapiDoc.ts

```diff
@@ -4,17 +4,17 @@ import type { OASDocument } from 'oas/types';
  * This is a snapshot of the OpenAPI description for ReadMe APIv2.
  *
  * This is used both for typechecking as well as for runtime validation.
- * We use  ajv to validate the user data against schemas in this document.
+ * We use ajv to validate the user data against schemas in this document.
  *
  * @see {@link https://docs.readme.com/main/openapi/readme-api-v2-beta.json}
  */
 const document = {
   openapi: '3.1.0',
   info: {
     description: 'Create beautiful product and API documentation with our developer friendly platform.',
-    version: '2.0.0-beta',
-    title: 'ReadMe API v2 🦉 (BETA)',
-    'x-readme-deploy': '5.344.0',
+    version: '2.0.0',
+    title: 'ReadMe API',
+    'x-readme-deploy': '5.395.0',
```

Comment:
the removal of the beta designation didn't happen in this deploy right?

Reply:
No it's happening in tomorrows release.